tier2_standards/release_procedure.md

```diff
@@ -0,0 +1,129 @@
+# Project Must Define and Document a Consistent Release Procedure
```

Comment:
I changed the name of the file and updated the link to it in the README because of a typo

Reply:
Thanks!